build/Dockerfile

```diff
@@ -29,14 +32,14 @@ ENV OPERATOR=/usr/local/bin/multicluster-operators-subscription \
     POLICY_GEN_ENABLE_HELM=true
 
 # install operator binary
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/multicluster-operators-subscription ${OPERATOR}
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/multicluster-operators-placementrule /usr/local/bin
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/uninstall-crd /usr/local/bin
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/appsubsummary /usr/local/bin
```

Comment:
The appsubsummary cli is required.

Reply:
Oops! Didn't mean to take that one out! Fixed!